.github/pull_request_template.md

```diff
@@ -0,0 +1,107 @@
+<!--
+You've got a Pull Request you want to submit? Awesome!
+This PR template is here to help ensure you're setup for success:
+  please fill it out to help ensure that your PR is complete and ready for approval.
+-->
+
+**JIRA Ticket:**
+[SOMEPROJECT-42](https://jira.cms.gov/browse/SOMEPROJECT-42)
+
+**User Story or Bug Summary:**
+<!-- Please copy-paste the brief user story or bug description that this PR is intended to address. -->
+
+
+### What Does This PR Do?
+
+<!--
+Add detailed discussion of changes here
+This is likely a summary, or the complete contents, of your commit messages.
```

Comment:
```suggestion
Add detailed description & discussion of changes here.
The contents of this section should be used as your commit message (unless you merge the PR via a merge commit, of course).

Please follow standard Git commit message guidelines:
* First line should be a capitalized, short (50 chars or less) summary.
* The rest of the message should be in standard Markdown format, wrapped to 72 characters.
* Describe your changes in imperative mood, e.g. "make xyzzy do frotz" instead of "[This patch] makes xyzzy do frotz" or "[I] changed xyzzy to do frotz", as if you are giving orders to the codebase to change its behavior.
* List all relevant Jira issue keys, one per line at the end of the message, per: <https://confluence.atlassian.com/jirasoftwarecloud/processing-issues-with-smart-commits-788960027.html>.

Reference: <https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project>.
```